bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/validation/ZeebeJobWorkerElementValidationTest.java

```diff
@@ -110,7 +110,6 @@ private BpmnModelInstance processWithJobWorkerElement(
   private static Stream<JobWorkerElementBuilder> jobWorkerElementBuilderProvider() {
     return Stream.of(
         JobWorkerElementBuilder.of("serviceTask", AbstractFlowNodeBuilder::serviceTask),
-        JobWorkerElementBuilder.of("businessRuleTask", AbstractFlowNodeBuilder::businessRuleTask),
```

Comment:
❓ This change is unclear to me. Why can't we test the business rule task any more as job worker element? It's still a job worker element as well right?

Reply:
Yes, it is. I removed the business rule task because of the test case `missingJobType()`. In the test, it verifies that the process has the violation `"Must have exactly one 'zeebe:taskDefinition' extension element")`. But this message is now different for business rule tasks because a business rule task can either have a `taskDefinition` or `calledDecision` extension.